templates/github/.github/workflows/nightly.yml.j2

```diff
@@ -66,6 +66,12 @@ jobs:
         run: |
           .github/workflows/scripts/before_script.sh
 
+      - name: Setting secrets
+        if: github.event_name != 'pull_request'
```

Comment:
You can probably remove this line.

Reply:
I believe removing it would break the CI, something similar broke pulp_ansible: https://github.com/pulp/pulp_ansible/runs/1444669298?check_suite_focus=true#step:8:9

Reply:
The build you link to is not a nightly build. I think that the code here in `nightly.yml` should not affect non-nightly builds?

Reply:
Also I don't think `github.event_name` would ever be `pull_request` here.

Reply:
true, it was a bad ctrl+c from ci.yml